src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -194,11 +196,109 @@ module OfficeInteropExtensions =
                                     |> Option.defaultValue ""
                                 )
                             )
+
                         ArcTable.fromStringSeqs(excelTable.name, headers, bodyRows)
                     )
                     return inMemoryTable
             }
 
+        /// <summary>
+        /// Checks whether the string seqs are part of a valid arc table or not and returns potential errors and index of them in the annotation table
+        /// </summary>
+        /// <param name="headers"></param>
+        /// <param name="rows"></param>
+        static member validateStringSeqs(headers:#seq<string>, rows:#seq<#seq<string>>) =
```

Comment:
Give this member a better name. The name should include the info, that the "stringSeqs" represent a Table.

Reply:
The problem with Spalte 3 is that it breaks a building block for Term and I was focussing on this. I was unsure how to validate otherwise and it seemed to be the most important part.

We could include more precise error warnings but I am using the ArcCtrl errors as a basis. For more precise warnings we would have to adapt or add functionality there when that is ok for you. I thought that might be a good thing for a future Issue. Now basic information that something is wrong and then mor precise information on the error in a future update.

Reply:
Oh okay I see,

Looking again into the code: I think, what's a bit confusing is that for every parsing error in the ARCTable you create two separate warnings.

Reply:
Yes, I keep the original error from ArcCtrl and then point to the column that creates the error. It looks better than when I pack both of them into one warning (at least my impression).

Reply:
fixed the naming of the method